newsletter/templates/newsletter/message/message.txt

```diff
@@ -12,4 +12,4 @@
 
 ++++++++++++++++++++
 
-Unsubscribe: http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}
+{% trans "Unsubscribe:" %} http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}

```

Comment:
Perhaps skip the `:`, that should save an extra string to translate. Most users are expecte to make custom templates anyway.